### PR TITLE
Ability to reference lambda layers by ARN

### PR DESCRIPTION
**Pull request by revmischa**
Normally you have to create a lambda `LayerVersion` object to reference an existing public lambda layer but I think it's more convenient to use a simple ARN string.

Lets me write 
```ts
const functionOptions: FunctionOptions = {
  bundling: {
    minify: true,
    sourceMap: true,
    externalModules: ["typeorm", "pg"], // from layer
  },
  layerArns: ["arn:aws:lambda:us-east-1:898466741470:layer:TypeORMPgLayerBD740AF5:3"],
}
```

Which is nice if you're really lazy

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
     "@aws-cdk/aws-apigatewayv2-integrations": "1.105.0",
     "@aws-cdk/aws-lambda": "1.105.0",
     "@aws-cdk/aws-lambda-nodejs": "1.105.0",
+    "@aws-cdk/aws-rds": "^1.107.0",
```
Comment (yvdlima) [inline package.json:57]: Just curious, was this import required for this PR?

**Review by yvdlima — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (revmischa) [reply package.json:57]: Probably not no

**Review by adamszeptycki — APPROVED** [review]
I would like to start working with this setup.

**Comment by revmischa** [comment]
> I would like to start working with this setup.

Please give it a go https://github.com/jetbridge/typescript-cdk-template https://github.com/jetbridge/freezy-backend

**Pull request merged**